it/controllers/pensions/incomeFromPensions/StatePensionLumpSumAmountControllerISpec.scala

```diff
@@ -0,0 +1,422 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.pensions.incomeFromPensions
+
+import builders.IncomeFromPensionsViewModelBuilder.anIncomeFromPensionsViewModel
+import builders.PensionsCYAModelBuilder.aPensionsCYAModel
+import builders.PensionsUserDataBuilder
+import builders.StateBenefitViewModelBuilder.anStateBenefitViewModelTwo
+import builders.UserBuilder.aUserRequest
+import forms.AmountForm
+import models.mongo.PensionsCYAModel
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.scalatest.BeforeAndAfterEach
+import play.api.http.HeaderNames
+import play.api.http.Status.{BAD_REQUEST, OK, SEE_OTHER}
+import play.api.libs.ws.WSResponse
+import utils.PageUrls.IncomeFromPensionsPages.statePensionLumpSumAmountUrl
+import utils.PageUrls.{fullUrl, pensionSummaryUrl}
+import utils.{IntegrationTest, PensionsDatabaseHelper, ViewHelpers}
+
+class StatePensionLumpSumAmountControllerISpec extends IntegrationTest with ViewHelpers with BeforeAndAfterEach with PensionsDatabaseHelper{
+
+  private val taxYearEOY: Int = taxYear - 1
+  private val poundPrefixText = "£"
+  private val amountInputName = "amount"
+
+  private def pensionsUsersData(isPrior: Boolean = false, pensionsCyaModel: PensionsCYAModel) = {
+    PensionsUserDataBuilder.aPensionsUserData.copy(isPriorSubmission = isPrior, pensions = pensionsCyaModel)
+  }
+
+  object Selectors {
+    val captionSelector: String = "#main-content > div > div > header > p"
+    val continueButtonSelector: String = "#continue"
+    val formSelector: String = "#main-content > div > div > form"
+    val hintTextSelector = "#amount-hint"
+    val poundPrefixSelector = ".govuk-input__prefix"
+    val inputSelector = "#amount"
+    val expectedErrorHref = "#amount"
+    val paragraphSelector: String = "#main-content > div > div > form > details > summary > span"
```

Comment:
Could rename this selector to something like details title selector to differentiate with the other paragraph selector